test/service/test_manifest_async.py

```diff
@@ -273,7 +272,8 @@ def test_execution_not_found(self, step_function_helper):
                 'Code': 'ExecutionDoesNotExist'
             }
         }, '')
-        response = requests.get(self.base_url + '/fetch/manifest/files', params=self.params)
+        response = requests.get(self.base_url.set(path='/fetch/manifest/files',
+                                                  args=self.params).url)
```

Comment:
@hannes-ucsc , I agreed with [Noah's advice](https://github.com/DataBiosphere/azul/pull/3011#discussion_r639310472), is it an acceptable change? It was done uniformly for similar invocations.